app/views/layouts/application.html.erb

```diff
@@ -9,12 +9,18 @@
   </head>
 
   <body>
-    <%= link_to 'Sign Up', new_users_path %>
-    <% if current_user %>
-      <%= link_to 'Logout', session_path(id: current_user.id), method: :delete %>
-    <% else %>
-      <%= link_to 'Login', new_session_path %>
-    <% end %>
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <ul class="nav navbar-nav">
+          <li><%= link_to 'Sign Up', new_users_path %></li>
+          <% if current_user %>
+            <li><%= link_to 'Logout', session_path(id: current_user.id), method: :delete %></li>
+          <% else %>
+            <li><%= link_to 'Login', new_session_path %></li>
+          <% end %>
```

Comment:
Should we move the sign-up bar below the else so it doesn't show up when a user is already active?